File: Client/Assets/iTweenExamples/Platforms/Scripts/CameraController.cs
using UnityEngine;
using System.Collections;

public class CameraController : MonoBehaviour
{
	bool hasFocus;
	
	public Transform target;
	
	void FixedUpdate ()
	{	
		Vector3 pos = target.position;
		pos.z=-14;
		pos.y=target.position.y+2f;
		iTween.MoveUpdate(gameObject,pos,.8f);
	}
	
	void OnGUI(){
		if(!hasFocus){
			if(GUI.Button(new Rect(75,70,100,50),"Start!")){
				hasFocus=true;
			}
		}
	}
}

File: Client/Assets/Package/Pixelplacement/iTweenExamples/Platforms/Scripts/CameraController.js
private var hasFocus : boolean;

public var target : Transform;

function FixedUpdate ()
{	
	var pos : Vector3  = target.position;
	pos.z=-14;
	pos.y=target.position.y+2;
	iTween.MoveUpdate(gameObject,pos,.8);
}

function OnGUI(){
	if(!hasFocus){
		if(GUI.Button(new Rect(75,70,100,50),"Start!")){
			hasFocus=true;
		}
	}
}